server/routes/auth_routes.py

```
@@ -155,6 +198,11 @@ def register():
                 record_audit_event(org_id, user_id, "register", "organization", org_id,
                                    {"email": email}, request)
 
+                try:
+                    send_verification_email(user_id, email)
+                except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)